static/templates/settings/bot-settings.handlebars

```diff
@@ -59,6 +59,13 @@
 
         <div class="form-horizontal" id="api_key_button_box">
             <div class="input-group side-padded-container">
+                <div>
+                    <span>Download single zuliprc of all active bots</span>
```

Comment:
Could the `zuliprc` name cause confusion because it's also used for other kinds of configuration files?

If so, would a name like `zulipbotrc` or `flaskbotrc` or (you can probably come up with something better) make sense?